src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int longSide;
+    private int shortSide;
+
+    public IsoscelesTrapezoid(String color, int height, int longSide, int shortSide) {
+        super(color);
+        this.height = height;
+        this.longSide = longSide;
+        this.shortSide = shortSide;
+    }
+
+    @Override
+    public double getArea() {
+        return ((height + longSide) / 2) * height;
```

Comment:
all arguments are int, you will have very imprecision result here

Reply:
Ok